core/src/main/scala/scalacss/package.scala

```diff
@@ -61,6 +65,13 @@ package object scalacss {
   case class CssKeyframesEntry(name  : KeyframeAnimationName,
                                frames: Map[KeyframeSelector, StyleStream]) extends CssEntry
 
+  case class CssFontFace(fontFamily  : String,
+                         src         : Seq[String],
```

Comment:
Yeah it seems strange that `src` can be empty but according to the syntax on [MDN](https://developer.mozilla.org/en/docs/Web/CSS/@font-face) it can be.

Ok... I just checked the [CSS3 spec](https://drafts.csswg.org/css-fonts-3/#font-face-rule).

> 4.3 Font reference: the src descriptor
> This descriptor specifies the resource containing font data. It is required for the @font-face rule to be valid.

That makes much more sense.